submissions/causal_machine_learning_in_healthcare.md

```diff
@@ -0,0 +1,420 @@
+
+# Introduction
+
+Causal inference is core to medicine. In this setting, we generally have
+some covariates (e.g., age, gender, images) about a patient and want to
+answer the counterfactual (Pearl and Mackenzie 2018) question: Which
+treatment would lead to the best outcome? The state of the art approach
+to answer this question are randomised controlled trials (RCTs) (Hariton
+and Locascio 2018) in which patients are assigned to the intervention or
+the comparator group at random. If the sample size is large enough, the
+act of randomization ensures that potential confounders (measured and
+unmeasured) are balanced between the groups which allows the attribution
+of differences in the outcome to the intervention. However, researchers
+face several challenges when conducting these trials:
+
+-   __Representativeness\/Generalisation__: The study is only applicable
+    to large groups in the real world if the original population of the
+    trial is representative. There may be biases in the population
+    (e.g., because of the recruitment process) that decrease
+    generalization.
+
+-   __Costs\/Resources__: RCTs are very expensive and require experts
+    and manual labor. In 2013, the average per-patient costs were
+    estimated at \$36,500 per trial phase and developing a new medicine
+    required an investment of around \$2.6 billion (“Biopharmaceutical
+    Industry-Sponsored Clinical Trials: Impact on State Economies”
+    2015).
+
+-   __Multiple Treatments__: We often want to compare more than one
+    treatment.
+
+-   __Measuring Outcomes__: For some diseases, measuring the outcome can
+    be hard, for instance because the effects are only observed after
+    some years.
+
+-   __Ethical Issues__: Not treating a patient can be unethical in some
+    settings.
+
+A second central application of causal inference in healthcare is the
+discovery of interventions that could be used as new treatment options.
+Currently, this is mainly done with experiments that are analyzed and
+visualized, leading to new insights and experiments to further refine a
+hypothesis. The problem with this approach is that it is manual and
+largely driven by domain experts: Someone needs to come up with good
+hypotheses, prioritise them, design the experiments, potentially merge
+the evidence with other experiments, and decide if the results are
+representative.
+
+A key challenge when applying causal inference techniques to healthcare
+is dealing with complexity. The causal generative process of the human
+body is very sophisticated and the causal relations span multiple scales
+of resolution, from reactions at the molecular level to symptoms of the
+body as a whole. Furthermore, because of the previously mentioned
+challenges with RCTs, addressing these problems by collecting more data
+is often not feasible.
+
+# Estimating Causal Treatment Effects from Observations
+
+## Framework
+
+We are in the Rubin-Neyman Potential Outcomes Framework (Rubin 2005)
```

Comment:
I think this section could use more discussion.  What exactly is the Rubin-Neyman Potential Outcomes Framework? Does it just describe ATE and ITE as suitable metrics for this problem? Are there alternative frameworks for measuring causality?